tests/testthat/test_rasterly.R

```diff
@@ -5,7 +5,8 @@ library(grid)
 library(plotly)
 library(rasterly)
 
-data <- data.table::data.table(x = rnorm(1e5), y = rnorm(1e5), z = sample(1:5, 1e5, replace = TRUE))
+data <- data.table::data.table(x = rnorm(1e5), y = rnorm(1e5), z = sample(1:5, 1e5, replace = TRUE), 
```

Comment:
As before, I'd replace `data` with an object name that does not correspond to a base R function name.

Reply:
Shouldn't be an issue. But it is no harm to replace it.

Reply:
It's up to you, but it's easy enough to change this to `sample_data` and avoid the collision -- it may be addressed by separate namespaces in the example, but we may not want to indirectly encourage this among users.